### ISSUE TITLE
Context root issue

### ISSUE THREAD

**Issue by chilmers** [algesten/fnug#30]
It seems like the javascript url inserted in the HTML is relative to the context root of the server  since it is pointing out for example /rs/mybundle/myfile.js which would mean something like http://localhost:9090/rs/mybundle/myfile.js. This creates a problem when you have a context path for your web app. If the webapp has a context path set to "client" then the javascript URL should be http://localhost:9090/client/rs/mybundle/myfile.js I guess.
I saw you are using getContextPath() to create a prefix, for some reason that doesn't seem to work for me, running on Jetty.
Something like this is generated:
&lt;script type="text/javascript" src="/rs/mybundle/_namespace.js"&gt;&lt;/script&gt;

BTW good work with this thing, looks very interesting!

**Comment by algesten** [comment]
I was pretty darn certain I fixed the context root problem. I just tested, and it appears to work. The project has the jetty:run plugin configured for debugging. I just tried altering the context root to 'client'.

In my HTML (notice I have relative URL, no context root here):
    <script type="text/javascript" src="rs/test.js"></script>

Which loads:
    <script type="text/javascript" src="/client/rs/test/bundle2-34d3c31312650a0d2235597991c89da2.js"></script>
    <link type="text/css" rel="stylesheet" href="/client/rs/test/test-67b54315466edaa9724f921399ae3fad.css">
    <script type="text/javascript" src="/client/rs/test/test-67b54315466edaa9724f921399ae3fad.js"></script>

Could you give me a test case? What OS?

**Comment by chilmers** [comment]
Sorry for taking so long, have been very busy lately.
Today I had some time to look at this again.

It´s on OS X.
I have configured bundles.js:
{
    simple: {
        basePath: '/js/',
        files: [ 
            'simple/namespace.js',  
            'simple/Engine.js',
            'simple/Car.js'
        ]
    }
}

and I have a reference to the code bundle on my page :

```
<script type="text/javascript" src="/client2/rs/simple.js"></script>
```

Where client2 is my apps context root.

I can see in firebug that a javascript gets injected which I guess should hold the bundle code but the src attribute lacks the context-path:

```
<script type="text/javascript" src="/rs/simple/simple-a4217e4964f0d01b0a9a8fb47cb13df.js" >
```

... which renders a 404 since it looks from the app server root instead of prepending with my apps context root (/client2)

I get the same result when using relative URL to reference the resource servlet as you say above, e.g.:
    <script type="text/javascript" src="rs/simple.js"></script>

**Comment by algesten** [comment]
Closed by 9f2550d28f9755a7cffa3202595c005a16fd0648 . Released in 0.26.
